### PR TITLE
Adding Style Transfer UI

### PR DESCRIPTION
**Pull request by ramongduraes**
User interface for the style transfer service.
Depends on #200 !

### CODE REVIEW

File: src/components/service/StyleTransfer.js [hunk 1]
```diff
@@ -0,0 +1,514 @@
...
+import InfoIcon from "@material-ui/icons/Info";
+import {createMuiTheme} from '@material-ui/core/styles';
+import Typography from "@material-ui/core/Typography";
+import Paper from "@material-ui/core/Paper";
+import grey from "@material-ui/core/es/colors/grey";
+import red from "@material-ui/core/es/colors/red";
+import Switch from "@material-ui/core/Switch";
+import Slider from "@material-ui/lab/Slider";
+
+
```
Comment (raamb) [inline src/components/service/StyleTransfer.js]: Any reason for the font to be Robot? The DApp uses muli. Would prefer to use that

File: src/components/service/StyleTransfer.js [hunk 2]
```diff
@@ -0,0 +1,514 @@
...
+        if (!this.state.isComplete) {
+            this.parseServiceSpec(nextProps.serviceSpec);
+        } else {
+            console.log(nextProps.response);
+            if (typeof nextProps.response !== 'undefined') {
+                this.state.response = nextProps.response;
+            }
+        }
+    }
+
```
Comment (raamb) [inline src/components/service/StyleTransfer.js]: Please refer to the #170 and the TemplateService.js. The new approach is a lot cleaner and reduces code in your component. Would be good to incorporate that

Comment (ramongduraes) [reply src/components/service/StyleTransfer.js]: Sorry about the sloppy code, I didn't notice you guys had an up-to-date version of the template. I believe its better now, please check again.

Comment (raamb) [reply src/components/service/StyleTransfer.js]: Not at all. The original example we provided was sloppy.

**Pull request merged**